pkg/server/registry/apigroups/acorn/volumes/translator.go

```diff
@@ -21,6 +21,36 @@ type Translator struct {
 	c kclient.Client
 }
 
+func (t *Translator) FromVolumeToPVName(ctx context.Context, namespace, name string) (string, error) {
+	i := strings.LastIndex(name, ".")
```

Comment:
This is a copy of `FromPublicName` although it tries to list PV with acorn volume api.